<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="mainHLayout">
      <item>
       <widget class="QLabel" name="mainLabel">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>36</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Welcome!</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="imgLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/basic/mainIcon.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="descriptionLabel">
      <property name="text">
       <string>&lt;h3&gt;Acesta e un mic IDE facut de Benta Andrei! &lt;br&gt; Nu e 100% functional dar o compilare de baza si producerea unui exe functioneaza.&lt;/h3&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_New">
     <property name="title">
      <string>&amp;New</string>
     </property>
     <addaction name="actionNew_File"/>
     <addaction name="actionNew_Project"/>
    </widget>
    <addaction name="menu_New"/>
    <addaction name="action_Open"/>
    <addaction name="actionSave_As"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Compile">
    <property name="title">
     <string>&amp;Compile</string>
    </property>
    <addaction name="action_Build"/>
    <addaction name="actionBuild_and_Run"/>
    <addaction name="actionRun"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionCompiler"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEdit"/>
   <addaction name="menu_Compile"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Build">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Build</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionBuild_and_Run">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Build and Run</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionNew_File">
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionCompiler">
   <property name="text">
    <string>Compiler</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
